dependencies/README.md

```diff
@@ -87,3 +89,80 @@ The tool allows users to choose the scope of entry point classes:
   - **All classes in the input class path**: when reachability check is off, then
     all static linkage errors from all classes in the classpath, regardless of the reachability,
     are reported.
+
+<a name="maven_dependency_graph"></a>
+## Maven Dependency Graph
+
+A Maven dependency graph is a graph data structure where
+- Node: a node is a Maven artifact identified as `groupId:artifactId[:classifier]:version`, where
+  `classifier` is optional.
+- Edge: an (directed) edge is a dependency between Maven artifacts. A dependency from a Maven
+  artifact (_source_ of the dependency) to another artifact (_target_ of the dependency) is defined
+  in the `dependencies` element in pom.xml of a Maven artifact.
+  
+  A dependency has a boolean attribute `optional` and a string attribute `scope`,
+  among other properties listed in [POM Reference: Dependencies][1].
+
+  Self-loops or parallel edges are not possible.
```

Comment:
Why not? Certainly this could be encoded in the XML. I think there are two cases here:

1. A self-loop is, I think, a compile time error and no graph will be constructed. 
2. A parallel edge is allowed but is dropped in the model.